content/manuals/docker-hub/repos/manage/hub-images/move.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Discover how to move images between repositories.
+keywords: Docker Hub, Hub, repository content, move
+title: Move images between repositories
+linkTitle: Move images
+weight: 40
+---
+
+Consolidating and organizing your Docker images across repositories can
+streamline your workflows, whether you're managing personal projects or
+contributing to an organization. This topic explains how to move images between
+Docker Hub repositories, ensuring that your content remains accessible and
+organized under the correct accounts or namespaces.
+
+## Personal to personal
+
+When consolidating personal repositories, you can pull private images from the initial repository and push them into another repository owned by you. To avoid losing your private images, perform the following steps:
+
+1. Navigate to [Docker Hub](https://hub.docker.com) create a new Docker account and select a personal subscription.
+2. Using `docker login` from the CLI, sign in using your original Docker account and pull your private images.
```

Comment:
should this step have a pull CLI command example like the following section does?